cmd/virt-handler/Dockerfile

```diff
@@ -23,9 +23,20 @@ FROM centos:7
 # no-lower than what was used when the binary was built.
 RUN yum -y install wget && \
     wget -P /etc/yum.repos.d https://copr.fedorainfracloud.org/coprs/jmliger/virt7-upstream/repo/epel-7/jmliger-virt7-upstream-epel-7.repo  \
-    && yum -y install libvirt-client \
+    && yum -y install libvirt-client cyrus-sasl-md5 \
```

Comment:
The username and secret are mounted inside the libvirt and the virt-handler container. The simplest solution, is to `kubectl exec` on a running virt-handler container. It already contains a properly configured client auth file. Just running `virsh` in there will work, just like it does right now, without authentication.

An example, how to view VMs:

```
kubectl exec virt-handler-z0ctc -- virsh list
 Id    Name                           State
----------------------------------------------------
 1     testns_testvm                  running
```

Using virsh interactive works like this:
```
kubectl exec -it virt-handler-z0ctc -- virsh
Welcome to virsh, the virtualization interactive terminal.

Type:  'help' for help with commands
       'quit' to quit

virsh # list                                                                                                                                                                            
 Id    Name                           State
----------------------------------------------------
 1     testns_testvm                  running

virsh # 
```